addons/l10n_ae/__manifest__.py

```diff
@@ -4,6 +4,7 @@
     'name': 'U.A.E. - Accounting',
     'author': 'Tech Receptives',
     'category': 'Accounting/Localizations/Account Charts',
+    'summary': 'UAE - Accounting',
```

Comment:
```suggestion
```
The `summary` field is not really meant to be used for search keywords,but it will be easier to find the module with its full name (see previous comment).
(same for `l10n_ae_pos`).